### ISSUE TITLE
Project status?

### ISSUE THREAD

**Issue by Gitoffthelawn** [bmatzelle/gow#237]
What is the project status?

Any recommended active forks?

**Comment by divinity76** [comment]
well, Cygwin does pretty much everything gow does, and a lot more. if Gow is lacking something you want, Cygwin probably has it. but you'll have to manually add `C:\cygwin64\bin` to your PATH system variable if you want to use the programs from cmd.
![envcyg](https://user-images.githubusercontent.com/1874996/28715899-9ac4900a-739a-11e7-978e-4097940b6029.gif)

**Comment by scardine** [comment]
The Windows Subsystem for Linux cover most of my use cases for GoW: now you can install Ubuntu over Windows directly from the Microsoft Store.

**Comment by Gitoffthelawn** [comment]
@scardine That's only for Windows 10, right?

**Comment by divinity76** [comment]
yup, but that said, Cygwin works from windows Vista and up, and does pretty much everything the win10 linux subsystem does, or so i've heard. (for what it's worth, i run windows 7, and my windows 10 running friends say they still prefer Cygwin, lol)

**Comment by scardine** [comment]
Windows 10, requires engaging the "Insider Preview" program last time I checked.

**Comment by T100D** [comment]
The advantage of GO is that it is small and easy to install on a windows server environment. I use it to do som quirks on windows system more easily for monitoring purposes. Love to see an updated version of Gow.

**Comment by lm8** [comment]
I build a lot of my utilities and libraries from source using MinGW.  I have a lot of the same programs built including later versions of the archiving utilities/libraries, curl and putty.  If there's interest, would be happy to share what I have.

**Comment by scardine** [comment]
> I build a lot of my utilities and libraries from source using MinGW. I have a lot of the same programs built including later versions of the archiving utilities/libraries, curl and putty. If there's interest, would be happy to share what I have.

Can you use them with Windows paths (`C:\Foo\Bar`) like in GoW? For me the greatest advantage of GoW over Cygwin is that you can use Windows paths instead of `/cygdrive/c/foo/bar`.

**Comment by lm8** [comment]
You can definitely use standard Windows paths.  Wouldn't be very useful in a Windows command prompt without them.  Later versions of Windows support use of slash or backslash interchangeably in most cases, so I've attempted to do the same.  I'm also not a big fan of the way Cygwin and msys show access to the file system.  I don't like abstractions like having /bin and /usr/bin pointing to the same location.  I'm slowly working towards getting enough native programs running so that I can build the programs I need (with GNU autotools or other build systems) without msys or Cygwin.  I have patches for some of the GNU autotools to get them to run natively.  They don't work properly if building natively on Windows with the code as is and their developers aren't interested in supporting native Windows or accepting patches to make their implementations work outside of Cygwin or msys environments on Windows..

**Comment by lm8** [comment]
I've uploaded my compilation of xz which doesn't appear to be in gow.  If there's interested, I can upload some other utilities not found in gow.  The link to find the upload is here;  https://lmemsm.dreamwidth.org/15650.html

**Comment by Gitoffthelawn** [comment]
@lm8 I'm interested. Can you put them in a GitHub repo? Thanks!

**Comment by lm8** [comment]
@Gitoffthelawn We can't access GitHub from work.  They block access.  So, no plans to add to a GitHub repository for now.

**Comment by Gitoffthelawn** [comment]
@lm8 They block GitHub?  LOL.  That's an odd choice of sites to block.  Regardless, I thank you for the update.
